.github/workflows/lint-format.yml

```diff
@@ -0,0 +1,35 @@
+name: "Lint & Format Code"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  format:
+    name: Format apply
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
```

Comment:
Think we should defer the toolchain to that defined in `rust-toolchain.toml`. It happens to be the same, but better to have this defined in one place. In other words, can we just remove the specification here.